### ISSUE TITLE
Feature request: Sound effect when recording begins

### ISSUE THREAD

**Issue by yioannides** [SeaDve/Kooha#313]
**Feature summary**

Some screencasting software have a sound when you begin a recording, for example a simple sine wave, a snapshot sound, a foley sound effect etc. It would be even more beneficial if there also was a countdown timer sound, like on macOS's builtin screenshot / screencasting tool. (that's a different topic, but it would be good to rename the Delay (Seconds) entry to Countdown)

This would make sense for people that want to begin recording on the window they want to record and have a semblance of time before starting, whereas now you have to guess when the recording starts.

If that's something that would be interested in implementing, I would be more than happy to provide you with sound effects of my creation.

**How would you like it to work**

This could be an on/off toggle option in the menu, a "Sound effects" option.

**Comment by SeaDve** [comment]
It was also actually implemented before, but removed because of the sound being recorded.

**Comment by yioannides** [comment]
That's a good point, I was thinking about that as well. I'm wondering if pipewire offers the ability to route a certain audio output only in monitor (speakers / headphones etc), but not in the computer's audio output. If not, I think a sound effect for the countdown could be a good alternative!

I think when global shortcuts land on GNOME, it'll be much easier to launch Kooha and start recording straight away in the window of your choice, without having to switch windows between the recording scene and Kooha, in order to avoid having to edit every screencast in Footage or Video Trimmer, for example.

**Comment by SeaDve** [comment]
> I think when global shortcuts land on GNOME, it'll be much easier to launch Kooha and start recording straight away in the window of your choice, without having to switch windows between the recording scene and Kooha, in order to avoid having to edit every screencast in Footage or Video Trimmer, for example.

Yep, that's nice to have indeed. I think there's already an open issue ticket for that.
